cfgen/cfgen

```diff
@@ -2149,6 +2294,10 @@ def build_cluster(cluster_desc: Dict[str, Any]) -> Cluster:
                           for aux_pool in aux_pools.get(pool_name, [])]
         ConfProfile.build(conf, root_id, prof)
 
+    if 'fdmi_filters' in cluster_desc:
+        for fdmi_filter in cluster_desc['fdmi_filters']:
```

Comment:
Here, the "cluster_desc['fdmi_filters']" might be None, because some cdf file may not have this defined.

Can we check if it is None?
```
    if 'fdmi_filters' in cluster_desc and cluster_desc['fdmi_filters'] is not None:
        for fdmi_filter in cluster_desc['fdmi_filters']:
```
Just like the same checking in other places?

I am not sure about this. So I am asking @max-seagate for help.

Reply:
I see similar code in:
```
 348 def validate_cluster_desc(desc: Dict[str, Any]) -> None:
 349     validate_nodes_desc(desc['nodes'])
 350     validate_pools_desc(desc['pools'])
 351     if 'profiles' in desc:
 352         validate_profiles_desc(desc['profiles'], desc['pools'])
 353     if 'fdmi_filters' in desc and desc['fdmi_filters'] is not None:
 354         validate_fdmi_filters_desc(desc['fdmi_filters'], desc['nodes'])
 355

```